src/ontology/mondo-edit.obo

```diff
@@ -544517,6 +544517,16 @@ def: "A group of epilepsy syndromes in which myoclonic seizures are a prominent
 synonym: "myoclonic epilepsy syndrome" EXACT [PMID:14641565]
 is_a: MONDO:0015650 {source="PMID:14641565"} ! epilepsy syndrome
 
+[Term]
+id: MONDO:0100580
+name: epilepsy, onset unknown
+def: "An epilepsy where the type of seizure onset is unknown or unclear if it is a generalized or focal epilepsy. Seizures of unknown onset may have features that can still be classified." [https://orcid.org/0000-0001-9310-0163, https://www.epilepsy.com/what-is-epilepsy/seizure-types, PMID:28276060]
```

Comment:
@nicolevasilevsky I have trouble understanding this sentence. Is the definition that we don't know the onset? (based on the term, this is correct). Does it include that we don't know the onset AND the generalized/focal type? 
I am suggesting a new definition, but please take a look to make sure that what we want to say is actually correct.
**Option 1:** 
An epilepsy in which the type of seizure onset is unknown and in which the generalized or focal epilepsy type is unclear. 

**Option 2:** 
An epilepsy in which the type of seizure onset is unknown or unclear, making it uncertain whether it is a generalized or focal epilepsy. Seizures of unknown onset may still have features that can be classified.

Reply:
thanks @sabrinatoro - these are great suggestions, I went with option #2